.github/CODEOWNERS

```diff
@@ -2,7 +2,7 @@
 /admin                                      @mukeshpanchal27 @felixarntz
 /tests/admin                                @mukeshpanchal27 @felixarntz
 /bin                                        @mukeshpanchal27 @felixarntz
-/.github                                    @mukeshpanchal27 @felixarntz
+/.github                                    @mukeshpanchal27 @felixarntz @thelovekesh
```

Comment:
Not opposed, but am wondering if we should specify code owners specifically for workflows, rather than the whole directory?

Reply:
@joemcgill Not sure what would be more appropriate but I am okay to review any workflow file changes.

Reply:
I think this would be more appropriate as I will be working on optimizing most of the workflows under https://github.com/WordPress/performance/issues/973